vhdbuilder/scripts/windows/collect-windows-logs.ps1

```diff
@@ -132,15 +132,6 @@ else {
   Write-Host "Calico logs not avalaible"
 }
 
-# log containerd containers (this is done for docker via networking collectlogs.ps1)
-$res = Get-Command ctr.exe -ErrorAction SilentlyContinue
```

Comment:
Will these be also collected by collectlogs.ps1 so we can remove it?

Reply:
Found it. I did not find the duplicate in the file but the file exists in your branch

Reply:
It is collected in L90-L104